Assets/Scripts/Enemy/EnemyPool.cs

```diff
@@ -79,7 +87,20 @@ public void Spawn(EnemyType et)
     {
         Events.EnemySpawned(et);
         var spawnPosition = ArenaGenerator.GetGridInstance().GetRandomSpawn();
-        PufferPool.Spawn(spawnPosition);
+
+        int r = (int)Mathf.Floor(Random.Range(0,3));

```

Comment:
p sure random.range gives you back an int
i'll slang it up all i want sux-qa fu